lib/roo_on_rails/sidekiq/process_scaling.rb

```diff
@@ -15,14 +15,15 @@ def initialize(queue_latencies)
 
       def current_processes
         ::Sidekiq::ProcessSet.new.count do |process|
+          process['quiet'] == 'false' &&
```

Comment:
Seems fair to to_s. @mezis I don't want to step on your toes mid merging; shall I do this or will you?